DfciPkg/IdentityAndAuthManager/UiSupport.c

```diff
@@ -365,7 +365,7 @@ UiEnrollRequest (
 
   CertText = QueryCertificateDetails (TrustedCert, TrustedCertSize);
   if (CertText == NULL) {
-    CertText    = AllocatePool (L'\0');
+    CertText    = AllocatePool (sizeof (L'\0'));

```

Comment:
Should we check that CertText is not NULL after this allocation before setting it?

Reply:
This was fixed by the CodeQL update - this is how I saw the allocation error.

Reply:
I added that link a few mins after this comment, but for anyone in the future, the exact PR being referenced is https://github.com/microsoft/mu_feature_dfci/pull/24.

As mentioned there, this was split out for future bisect since it has a higher likelihood of changing behavior (in a positive way) than the issues actually found by CodeQL.